app/v3/amendPeriodSummary/AmendPeriodSummaryController.scala

```diff
@@ -42,8 +42,10 @@ class AmendPeriodSummaryController @Inject() (val authService: EnrolmentsAuthSer
                                               idGenerator: IdGenerator)(implicit ec: ExecutionContext, appConfig: AppConfig)
     extends AuthorisedController(cc) {
 
+  val endpointName = "amend-period-summary"
+
   implicit val endpointLogContext: EndpointLogContext =
-    EndpointLogContext(controllerName = "AmendPeriodSummaryController", endpointName = "amendSelfEmploymentPeriodSummary")
+    EndpointLogContext(controllerName = "AmendPeriodSummaryController", endpointName = endpointName)
```

Comment:
I think the log-context names need to be unchanged, as the support/live services teams create log filters etc based on them (I think). Just safer to keep them as they were in case we're breaking something.